recipes/config.rb

```diff
@@ -27,6 +27,56 @@
   recursive true
 end
 
+directory '/etc/default' do
+  mode '0755'
+  owner 'root'
+  group 'root'
+  action :create
+  recursive true
+end
+
+# Setup /etc/default/impala
+if node['impala'].key?('config')
+  my_vars = { :options => node['impala']['config'] }
+
+  impala_log_dir =
+    if node['impala']['config'].key?('impala_log_dir')
```

Comment:
So in the attribute file, they should look like:
default['impala']['config']['impala_catalog_service_host'] = '127.0.0.1'
and this: templates/default/generic-env.sh.erb:  export <%= key.upcase %>="<%= value %>"
converts them to upper case in the resulting chef run, right?